tests/installation/installation_overview.pm

```diff
@@ -69,6 +69,18 @@ sub run {
         $self->deal_with_dependency_issues;
         assert_screen "inst-xen-pattern" if get_var('XEN');
         ensure_ssh_unblocked;
+        # Check the systemd target, see poo#45020
+        if (get_var('DESKTOP')) {
+            # QAM do some installs with DESKTOP=textmode, but install gnome... 
+            # and switching between ttys does not work for hyperv.
+            return if (get_var(QAM_MINIMAL) || check_var('MACHINE', 'svirt-hyperv'));
```

Comment:
you can move early `return` and related comments before `if`.

Reply:
Done.